### ISSUE TITLE
NullPointerException when streaming es -> es

### ISSUE THREAD

**Issue by tylerjl** [elasticsearch/stream2es#41]
I'm moving some small indices (~1-2k documents) to another index and got this stack trace on one of them:

```
java.lang.NullPointerException
	at java.util.regex.Matcher.getTextLength(Matcher.java:1234)
	at java.util.regex.Matcher.reset(Matcher.java:308)
	at java.util.regex.Matcher.<init>(Matcher.java:228)
	at java.util.regex.Pattern.matcher(Pattern.java:1088)
	at clojure.core$re_matcher.invoke(core.clj:4386)
	at clojure.core$re_find.invoke(core.clj:4438)
	at stream2es.es$scroll_STAR_.invoke(es.clj:72)
	at stream2es.es$scroll.invoke(es.clj:79)
	at stream2es.es$scan.invoke(es.clj:103)
	at stream2es.stream.es$make_callback$fn__3008.invoke(es.clj:75)
	at stream2es.main$stream_BANG_.invoke(main.clj:241)
	at stream2es.main$main.invoke(main.clj:330)
	at stream2es.main$_main.doInvoke(main.clj:336)
	at clojure.lang.RestFn.applyTo(RestFn.java:137)
	at stream2es.main.main(Unknown Source)
2015-01-19T19:59:12.461+0000 ERROR unexpected exception: java.lang.NullPointerException
2015-01-19T19:59:12.633+0000 INFO  00:07.973 0.0d/s 0.0K/s (0.0mb) indexed 0 streamed 0 errors 0
```

Unfortunately I'm doing these in large batches so I can't pin down the exact index/document for debugging.

Running on version:

    2015-01-19T20:16:11.716+0000 INFO  stream2es 2014122282ace27

**Comment by jonahbull** [comment]
@tylerjl Looks you're running into #40.
Any further thoughts on that @drewr ?

**Comment by tylerjl** [comment]
Yep, confirmed that lowering scroll size fixes this - closing as a duplicate of #40
